docs/k8s.md

```diff
@@ -0,0 +1,79 @@
+# Konveyor Forklift k8s Installation Instructions
+
+## Pre-requisites
+
+- **Kubernetes cluster or Minikube v1.17+**
+- **OLM**
+
+### Installing OLM support
+
+We strongly suggest OLM support for Forklift deployments, in some production kubernetes clusters OLM might already be present, if not, follow official instructions in how to add it. Below is an example in how to add it to minikube.
+
+`minikube addons enable olm`
+
```

Comment:
Can you also link to the OLM documentation for the deployment on vanilla k8s? And if it's really short, maybe add the steps directly here.

Reply:
Done , see last commit